docs/source/getting_started.rst

```diff
@@ -117,57 +117,70 @@ To create a satellite source, use the following steps:
 
 To run a scan, use the following steps:
 
-Run the scan by using the ``scan start`` command, specifying one or more sources for the ``sources`` option:
+Run the scan by using the ``scan start`` command, specifying the name of a scan for the ``name`` option:
 
-  ``# qpc scan start --sources source_name1 source_name2``
+  ``# qpc scan start --name scan_name1``
 
-For example, if you want to scan the network source ``mynetwork`` and the Satellite source ``mysatellite6``, you would enter the following command:
+For example, if you want to run the scan ``myscan``, you would enter the following command:
 
-  ``# qpc scan start --sources mynetwork mysatellite6``
+  ``# qpc scan start --name myscan``
 
-Showing Scan Results for an Active Scan
----------------------------------------
-When you run the ``scan start`` command, the output provides an identifier for that scan. You can show the scan results to follow the status of the scan by using the ``scan show`` command and specifying the provided identifier.
+Showing Scan Job Results for an Active Scan
+-------------------------------------------
+When you run the ``scan start`` command, the output provides an identifier for that scan job. You can show the scan job results to follow the status of the scan job by using the ``scan job`` command and specifying the provided identifier.
 
-**IMPORTANT:** The ``scan show`` command can show results only after the scan starts running. You can also use this command on a scan that is completed.
+**IMPORTANT:** The ``scan job`` command can show results only after the scan job starts running. You can also use this command on a scan job that is completed.
 
 For example, you could run the following scan as the first scan in your environment:
 
-  ``# qpc scan start --sources mynetwork mysatellite6``
+  ``# qpc scan start --name myscan``
 
-The output for the command shows the following information, with ``1`` listed as the scan identifier.
+The output for the command shows the following information, with ``1`` listed as the scan job identifier.
 
   ``Scan "1" started``
 
 To show the scan results to follow the status of that scan, you would enter the following command:
 
-  ``# qpc scan show --id 1``
+  ``# qpc scan job --id 1``
 
 Listing Scan Results
 --------------------
-In addition to showing the status of a single scan, you can also show a list of all scans that are in progress or are completed on the Quipucords server. To show this list of scans, you use the ``scan list`` command. The output of this command includes the scan identifier, the source or sources for that scan, and the current state of the scan.
+In addition to showing the status of a single scan job, you can also show a list of all scans that are in progress or are completed for a particular scan. To show this list of scan jobs, you use the ``scan job`` command. The output of this command includes the scan job identifier, the source or sources for that scan, and the current state of the scan.
```

Comment:
Should we say `you can also show a list of all jobs that are in progress`

Reply:
done